examples/solid-contacts/src/client/stores/contacts.ts

```diff
@@ -0,0 +1,72 @@
+﻿import { isServer } from 'solid-js/web'
+import { IContact } from '~/server/database/entities/contact'
+
+const store = (isServer as boolean) ? undefined : createStore()
```

Comment:
This is a general question, and I think my only important comment in this review.

`createStore` is also the name of a function from `solid-js/store`, and is an implementation of deep reactivity. Assuming it won't be used here, we should probably rename the store(+function) here to some other names, so as not to cause confusion. However, maybe we should be using stores? Or rather, can we even use a store here? I have no idea how it works. I see the [docs say](https://start.solidjs.com/api/createRouteData#setting-the-reconcile-key) that createRouteData internally stores data in a store, but does that mean that we are doing fine grained updates when we update data?

If you know the answer to this question, great. Otherwise, we'll ping the solid-start team to get an answer.

Reply:
The intention was to apply what is called "the store pattern" in Vue.js which is meant for simplified global state management. It seems my habit of naming global state as "store" not only conflicts with Solid's primitives, but it also ignores the fact that the state here is not meant to be reactive since `routeData` will pull a new copy of it at every navigation. Renamed to cache.